### PR TITLE
Xenial: Update to the latest stable ruby versions 

### PR DESCRIPTION
**Pull request by DirtyF**
Fix #178 

```
In order to benefit from the latest security fixes
As a worried user
I can run latest Ruby patches
```

Details:

- https://www.ruby-lang.org/en/news/2018/10/17/ruby-2-4-5-released/
- https://www.ruby-lang.org/en/news/2018/10/17/ruby-2-3-8-released/
- https://www.ruby-lang.org/en/news/2018/03/28/ruby-2-2-10-released/

### CODE REVIEW

**Comment by bcomnes** [comment]
Thanks for the contribution! I'll be bumping and merging these shortly.

**Comment by bcomnes** [comment]
I'm going to tweak this a little bit, and also backport it to the trusty image.

**Comment by bcomnes** [comment]
Actually, I'm going to merge this to the trusty image, and do something similar, but slightly different for the xenial image.

**Comment by bcomnes** [comment]
Actually, nevermind, this was rebased or something that prevents that.  No problem, Ill modify this one for the xenial image.

File: Dockerfile [hunk 1]
```diff
@@ -226,16 +226,19 @@ RUN adduser --system --disabled-password --uid 2500 --quiet buildbot --home /opt
...
 ENV PATH /usr/local/rvm/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
 
+# Match this set latest Stable releases we can support on https://www.ruby-lang.org/en/downloads/
 RUN /bin/bash -c "source ~/.rvm/scripts/rvm && \
-                  rvm install 2.3.8 && rvm use 2.3.8 && gem install bundler && \
+                  rvm install 2.6.1 && rvm use 2.6.1 && gem install bundler && \
+                  rvm install 2.5.3 && rvm use 2.5.3 && gem install bundler && \
                   rvm install 2.4.5 && rvm use 2.4.5 && gem install bundler && \
-                  rvm use 2.3.8 --default && rvm cleanup all"
+                  rvm use 2.6.1 --default && rvm cleanup all"
```
Comment (DirtyF) [inline Dockerfile]: 😍

Comment (bcomnes) [reply Dockerfile]: For the xenial image only a this point, What do you think about bumping the default on the trusty image to 2.5.3?  Was going to do that today as well.

Comment (DirtyF) [reply Dockerfile]: The more recent, the more faster builds, GitHub Pages are currently using 2.5.3: https://pages.github.com/versions/

**Pull request merged**